lib/linguist/languages.yml

```diff
@@ -6873,6 +6873,18 @@ YAML:
   codemirror_mode: yaml
   codemirror_mime_type: text/x-yaml
   language_id: 407
+YAML+ERB:
+  type: markup
+  color: "#cb171e"
+  tm_scope: source.yaml.erb
```

Comment:
We don't have a grammar which has this scope. 

You need to either add such a grammar as part of this PR or change this to match a scope which Linguist already knows about which has support for YAML+ERB syntax.